### DISCUSSION TITLE
How to use description tag only for SEO

### DISCUSSION THREAD

**Discussion by gusibi** [gethinode/hinode#792] [Help] [answered]
Hello, I would like to inquire about something. If I want to use the set 'description' tag only for SEO purposes and not display it on the page, is there any configuration that can be used?
![image](https://github.com/gethinode/hinode/assets/1282277/c882e716-15ae-4283-b7d5-549072ca5acb)

**Comment by markdumay** [comment, marked as answer]
You can customize the rendering of pages in `layouts/_default`. Single pages use `layouts/_default/single.html` as base template. It includes a reference to `layouts/_default/single/header.html`, which renders the description. You can delete this line to remove the description from all single pages:

```html
<p class="lead mb-5 mt-3">{{ .Description }}</p>
```

Alternatively, you can define a custom layout type such as `mylayout`. Copy the file `layouts/_default/single/header.html` to `layouts/mylayout/single/header.html` and remove the description. Now set `type: mylayout` in the frontmatter of the markdown file you want to customize.

**Comment by gusibi** [comment]
It was a success, thank you very much.

**Comment by ansariseo** [comment]
In an SEO course at[ Ideocollege Institute ](https://ideocollege.com/seo-course/)or any reputable institution, you would learn how to optimize meta description tags for SEO purposes. Here's how to effectively use the description tag for SEO:
Relevance and Keyword Placement:
Ensure that the meta description accurately reflects the content of the page.
Incorporate relevant keywords naturally within the description to signal to search engines what the page is about.
Compelling and Engaging Content:
Write a compelling and engaging meta description that entices users to click through to your website.
Highlight the unique value proposition or key benefits of the page to encourage user engagement.
Length and Formatting:
Keep the meta description concise, typically between 150 to 160 characters, to ensure it displays properly in search engine results.
Use clear and concise language to convey the main message of the page and avoid unnecessary fluff.
Call to Action (CTA):
Include a clear call to action (CTA) in the meta description to prompt users to take action, such as "Learn More," "Shop Now," or "Sign Up Today."
A compelling CTA can increase click-through rates (CTR) and improve the overall effectiveness of the meta description.
Unique and Descriptive:
Avoid duplicate meta descriptions across multiple pages of your website. Each meta description should be unique and tailored to the specific content of the page.
Provide a concise summary of what users can expect to find on the page, focusing on the most important and relevant information.
Testing and Optimization:
Regularly review and test different variations of meta descriptions to identify which ones perform best in terms of CTR and user engagement.
Use A/B testing or SEO tools to compare the effectiveness of different meta descriptions and optimize accordingly.
By following these guidelines and applying the principles taught in an SEO course at [Ideocollege Institute, ](https://ideocollege.com/seo-course/)you can effectively use meta description tags to improve your website's visibility in search engine results and attract more organic traffic.
